tableauserverclient/server/server.py

```diff
@@ -30,7 +31,7 @@ def __init__(self, server_address, use_server_version=False):
         self._session = requests.Session()
         self._http_options = dict()
 
-        self.version = "2.3"
+        self.version = "3.0"
```

Comment:
Don't change this one either.  2.3 is the older version we support, so we want to default to the oldest version unless specifically detected to be earlier.

Reply:
Hi Russ,

So if I remove this change and initialise the server with use_server_version = True. then it works. Is that how I'm supposed to do it?

Do we expect people to know that or is it documented?
Marc